theme/saas/static/js/script.js

```diff
@@ -0,0 +1,40 @@
+var client = algoliasearch('JD256SSXSD', '645cadc9c306b19f70bac41e09ad2ecc');
+var index = client.initIndex('shippableRCDocs');
+$('#shippable-search').autocomplete({ hint: true, openOnFocus: true }, [
```

Comment:
Move this to a var called autocompleteOptions. Add comments on why they are turned on.